README.adoc

```diff
@@ -41,13 +41,20 @@ endif::[]
 
 The Vulkan Samples is collection of resources to help you develop optimized Vulkan applications.
 
-If you are new to Vulkan the link:samples/api[API samples] are the right place to start.
+If you are new to Vulkan the xref:samples/api/README.adoc[API samples] are the right place to start.
 Additionally you may find the following links useful:
 
+ifdef::site-gen-antora[]
+* xref:guide:ROOT:index.adoc[Vulkan Guide]
+* xref:tutorial:ROOT:00_Introduction.adoc[Get Started in Vulkan]
+endif::[]
+
+ifndef::site-gen-antora[]
```

Comment:
It seems, there's nothing like an `else::[]`?

Reply:
Exactly. Asciidoc doesn't have an else for defines, see https://docs.asciidoctor.org/asciidoc/latest/directives/ifdef-ifndef/